src/components/ModalFooter/ModalFooter.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC, ReactElement } from "react";
+import cx from "classnames";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import classes from "./ModalFooter.module.scss";
+
+interface ModalFooterProps extends VibeComponentProps {
+  children: ReactElement | ReactElement[] | string;
```

Comment:
@SergeyRoyt Please add task to create a unified children type so would could use it everywhere?

Reply:
Yes, was thinking about that too. Will do